props/proptypes.md

```diff
@@ -0,0 +1,226 @@
+# PropTypes
+
+## Goals
+
+- Define `PropTypes`
+
+## Introduction
+
+Sofia created an attendance app that displays a list of students using a `Student` component.
+
+<!-- prettier-ignore-start -->
+```js
+const Student = (props) => {
+    return (
+        <li>
+            <ul>
+                <li>Nickname: {props.name}</li>
+                <li>Email: {props.email}</li>
+            </ul>
+        </li>
+    )
+}
+```
+<!-- prettier-ignore-end -->
+
+However, when another component renders a `Student` component, but doesn't pass in the `name` and `email` properties, our `Student` component looks totally broken!
+
+<!-- prettier-ignore-start -->
+```js
+const StudentList = () => {
+    return (
+        <section>
+                <Student></Student>
+                <Student name='Soo-ah' email='sooah@dev.org'></Student>
+        </section>
+    )
+}
+```
+<!-- prettier-ignore-end -->
+
+![An attendance web app with a list of two students. One student reads "Nickname: , Email: ." The second student reads "Nickname: Soo-ah, Email: sooah@dev.org"](../assets/props_proptypes_undefined.png)  
+__Fig. An attendance web app with a list of two students. One student reads "Nickname: , Email: ." The second student reads "Nickname: Soo-ah, Email: sooah@dev.org"_
+
+It would be nice if the React library could enforce the presence or data type of our props...
+
+## PropTypes
+
+**PropTypes** is a React feature that allows us to explicitly list the props a component takes and their types.
+
+When we use PropTypes effectively, PropTypes will:
+
+- Provide type-checking and validation of props
+- Create implied self-documentation of our components
+
+Both of these benefit our developer workflow and communication!
+
+## PropType Syntax
+
+We define PropTypes per component.
+
+For each component, we need to do the following:
+
+1. Import `PropTypes` into our component file
+1. Attach a `propTypes` member to our component function
+1. Assign the `propTypes` member to an object, where each key-value pair is a `prop` name and a _validator_
```

Comment:
directionality of assign
```suggestion
1. Assign an object to the `propTypes` member, where each key-value pair in the object is a `prop` name and a _validator_
```

Reply:
Dangit!!!!!

Simon: 0. The word "assign": 40928348234